src/foam/nanos/dig/DIG.js

```diff
@@ -53,6 +53,18 @@ foam.CLASS({
     },
     'cmd',
     'format',
+    {
+      name: 'digColumns',
+      expression: function(daoKey) {
+         var dao = this.__context__[daoKey];
+
+         if ( dao.of.getAxiomByName('digColumns') ) {
+          return dao.of.getAxiomByName('digColumns').columns.toString();
```

Comment:
Remove this for now. We'll introduce the `digColumns` axiom when we're more ready later.